### PR TITLE
[Improvement] Remove aws bundle dependencies from ams bin

### PR DESCRIPTION
**Pull request by zhoujinsong**
## Why are the changes needed?

Exclude aws bundles from hadoop-aws, we have declared the dependencies of AWS separately.

## Brief change log

- Exclude aws bundles from hadoop-aws

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (no)
- If yes, how is the feature documented? (not applicable)

### CODE REVIEW

**Comment by zhoujinsong** [comment]
I ran the quickstart with minio and it worked fine.

**Review by zhouxinyu — APPROVED** [review]
LGTM~

**Review by XBaith — APPROVED** [review]
LGTM

File: pom.xml [hunk 1]
```diff
@@ -400,6 +400,17 @@
...
                 <version>${hadoop.version}</version>
+                <!-- Exclude aws bundles, we have declared the dependencies of AWS separately. -->
+                <exclusions>
+                    <exclusion>
+                        <groupId>software.amazon.awssdk</groupId>
+                        <artifactId>bundle</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>com.amazonaws</groupId>
+                        <artifactId>aws-java-sdk-bundle</artifactId>
```
Comment (xleoken) [inline pom.xml:411]: it seems doesn't contains `aws-java-sdk-bundle` module

image

**Comment by zhoujinsong** [comment]
Thanks for the review. @zhouxinyu @XBaith

**Pull request merged**